Task 1:
1. Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A
researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose
levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose
level of 108. Test the hypothesis that the raw cornstarch had an effect or not.

To test the hypothesis 

Null hypothesis- There is a effect of cornstarch has effect ofsugar level

H0 $\mu$= 100

Alternate hypothesis-There is no effect.

H1 $\mu$>100

There is no significance level given so we will take 5% ie 0.05

Standard Deviation of sample=standard deviation/sqrt(sample size)

Sample Size SS=36

In [51]:
import math

In [52]:
x=108
u=100
SS=(15/ math.sqrt(36))
z=(x-u)/SS

In [53]:
z

3.2

Seeing the ztable p value for 3.20=0.993

So this the probability that value will be less than 108

So probailty that the null hypothesis is correct
1-0.993

In [54]:
z0=1-0.993

In [55]:
z0

0.007000000000000006

Since 0.007(Probabilty of hypothesis to be true) is less than significance value 0.05.

So we will reject the null hypothesis i.e cornstarch has any effect


2. In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second state,
47% of the voters are Republicans, and 53% are Democrats. Suppose a simple random sample
of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican voters in
the second state than in the first state?

pr1- Portion of Republican voter in 1st state

pr2=portion of Rebublican voters in 2nd state

n1=Sample size of 1st state

n2=Sample size of 2nd state

First we will see if sample size is enough.

In [56]:
pr1=.52
pr2=.47
n1=100
n2=100
print(n1*pr1)
print(n1*(1-pr1))
print(n2*pr2)
print(n2*(1-pr2))


52.0
48.0
47.0
53.0


Since all the values are greater than 10, so the sample size is enogh.

Mean difference of portions.


In [57]:
E=pr1-pr2
E

0.050000000000000044

standard deviation of difference

In [58]:
sd=math.sqrt(((pr1*(1-pr1))/n1)+((pr2*(1-pr2))/n2))
sd

0.07061869440877536

probability that the survey will show a greater percentage of Republican voters in
the second state than in the first state.
So we will find the probability of pr1-pr2<0


X=0


In [59]:
X=0
z=(X-E)/sd
z

-0.7080278164104213

In [60]:
pval=.2420



So the probability that the survey will show a greater percentage of Republican voters in
the second state than in the first state=.2420

3. You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

X=Score of 1100

u=Mean test score

In [61]:
X=1100
u=1026
st=209

In [62]:
zst=(X-u)/st
zst

0.35406698564593303

So my score is 0.354 standard deviation from the mean.

Task 2:
1. Is gender independent of education level? A random sample of 395 people were surveyed
and each person was asked to report the highest education level they obtained. The data that
resulted from the survey is summarized in the following table:

        High School Bachelors Masters Ph.d. Total

Female   60 54 46 41 201

Male    40 44 53 57 194

Total   100 98 99 98 395

Question: Are gender and education level dependent at 5% level of significance? In other

words, given the data collected above, is there a relationship between the gender of an

individual and the level of education that they have obtained?

null hyp

In [63]:
n=((60-50.886)**2)/50.866+((54-49.868)**2)/49.868+((46-50.377)**2)/50.377+((41-49.868)**2)/49.868+((40-49.114)**2)/49.114+((44-48.132)**2)/48.132+((53-48.623)**2)/48.623+((57-48.132)**2)/48.132
n

8.006548954258284

The critical value of n is 7.815
since 8.0065>7.815
So we conclude that education levels depend on gender.

2. Using the following data, perform a oneway analysis of variance using α=.05. Write up the
results in APA format.

[Group1: 51, 45, 33, 45, 67]

[Group2: 23, 43, 23, 43, 45]

[Group3: 56, 76, 74, 87, 56]

In [64]:
gp1=[51, 45, 33, 45, 67]
gp2=[23, 43, 23, 43, 45]
gp3=[56, 76, 74, 87, 56]

Mean of all groups-

In [65]:
mngp1=sum(gp1)/len(gp1)
mngp1

48.2

Sum of suare of deviation-

In [66]:
sdgp1=(gp1[0]-mngp1)**2+(gp1[1]-mngp1)**2+(gp1[2]-mngp1)**2+(gp1[3]-mngp1)**2+(gp1[4]-mngp1)**2
sdgp1

612.8

In [67]:
mngp2=sum(gp2)/len(gp2)
mngp2

35.4

Sum of suare of deviation-

In [68]:
sdgp2=(gp2[0]-mngp2)**2+(gp2[1]-mngp2)**2+(gp2[2]-mngp2)**2+(gp2[3]-mngp2)**2+(gp2[4]-mngp2)**2
sdgp2

515.2

In [69]:
mngp3=sum(gp3)/len(gp3)
mngp3

69.8

Sum of suare of deviation-

In [70]:
sdgp3=(gp3[0]-mngp3)**2+(gp3[1]-mngp3)**2+(gp3[2]-mngp3)**2+(gp3[3]-mngp3)**2+(gp3[4]-mngp3)**2
sdgp3

732.8

sum of all sqaure of devation -

In [71]:
n=len(gp1)-1

In [72]:
var1=sdgp1/n
var2=sdgp2/n
var3=sdgp3/n
print(var1,var2,var3)

153.2 128.8 183.2


In [73]:
MSer=(var1+var2+var3)/3
MSer

155.06666666666666

Error remaining-

In [74]:
remer=MSer*((len(gp1)+len(gp2)+len(gp3)-3))
remer

1860.8

Grand Mean-

In [75]:
gnmn=(mngp1+mngp2+mngp3)/3
gnmn

51.133333333333326

Using grand mean and group mean the square of deviation is-

In [76]:
sdgn=((mngp1-gnmn)**2+(mngp2-gnmn)**2+(mngp3-gnmn)**2)
sdgn

604.5866666666666

In [77]:
varmns=sdgn/(3-1)
varmns

302.2933333333333

In [78]:
msbt=varmns*5
msbt

1511.4666666666665

In [79]:
dfgroups=3-1 

SSgroup=(1511.45)*dfgroups

print(dfgroups,SSgroup)



2 3022.9


Test statistic and critical value

In [80]:
F=msbt/MSer

Fcr=3.89


print(F,Fcr)

9.747205503009457 3.89


ANOVA table

source	  SS	   df	MS	    F

group	 3022.9	  2	  1511.45	9.75

error	 1860.8	  12 155.07	

total	 4883.7			

Effect size

In [83]:
n2=SSgroup/(SSgroup+remer)
n2

0.6189774146651105

APA writeup
F(2, 12)=9.75, p <0.05, η2=0.62.

3. Calculate F Test for given 10, 20, 30, 40, 50 and 5,10,15, 20, 25.

For 10, 20, 30, 40, 50:



In [84]:
input1=(10,20,30,40,50)

print(len(input1))

mn=sum(input1)/len(input1)
print(mn)

5
30.0


In [85]:
sum(input1)

150

standard Deviation

In [86]:


sd1=math.sqrt(((input1[0]-mn)**2+(input1[1]-mn)**2+(input1[2]-mn)**2+(input1[3]-mn)**2+(input1[4]-mn)**2)/(len(input1)))
sd1

14.142135623730951

In [87]:
variance1=sd1**2
variance1

200.00000000000003

For 2nd input

In [88]:
input2=(5,10,15, 20, 25)

In [89]:
print(len(input2))
mn2=sum(input2)/len(input2)
print(mn2)

5
15.0


In [90]:
sd2=math.sqrt(((input2[0]-mn2)**2+(input2[1]-mn2)**2+(input2[2]-mn2)**2+(input2[3]-mn2)**2+(input2[4]-mn2)**2)/len(input2))

In [91]:
variance2=sd2**2
variance2

50.00000000000001

F Test= (variance of 10, 20,30,40,50) / (variance of 5, 10, 15, 20, 25) 

In [92]:
ftest=variance1/variance2
ftest

4.0